UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Blob/BlobStructure.cs

```diff
@@ -48,9 +47,22 @@ public class BlobStructure : MonoBehaviour
 
 		private bool initialSet;
 
+		[HideInInspector]
+		public bool connectedToBlobNet;
+
+		[HideInInspector]
+		public BlobStructure connectedNode;
+
+		[HideInInspector]
+		public List<Vector3Int> connectedPath = new List<Vector3Int>();
+
+		[HideInInspector]
+		public LineRenderer lineRenderer;
+
 		private void OnEnable()
 		{
 			integrity = GetComponent<Integrity>();
+			lineRenderer = GetComponent<LineRenderer>();
```

Comment:
any reason to cache the components OnEnable instead of on Awake like most scripts do?

Reply:
Dont think so, ive changed it over to use awake instead